src/sentry/api/urls.py

```diff
@@ -412,6 +413,18 @@
         name='sentry-api-0-user-subscriptions'
     ),
 
+    # Incidents
+    url(
+        r'^organizations/(?P<organization_slug>[^\/]+)/incidents/$',
+        OrganizationIncidentIndexEndpoint.as_view(),
+        name='sentry-api-0-organization-incident-index'
+    ),
+    url(
+        r'^incidents/(?P<incident_id>[^\/]+)/$',
```

Comment:
Should this be `/organizations/:orgid/incidents/:id`? to match other resources like releases and user-feedback?

Reply:
Yeah, I wasn't sure since I've seen examples of doing it both ways, but it probably makes sense.